xblock/mixins.py

```diff
@@ -447,3 +447,18 @@ def _add_field(self, node, field_name, field):
             node.insert(0, elem)
         else:
             node.set(field_name, unicode(field.read_from(self)))
+
+
+class IndexInfoMixin(object):
+    """
+    This mixin provides interface for classes that wish to provide index
+    information which might be used within a search index
+    """
+
+    def index_view(self):
```

Comment:
I would probably call this something other than `_view`, since it doesn't return a `Fragment` (unlike other XBlock views)p